### ISSUE TITLE
Exponential sentry error suppression

### ISSUE THREAD

**Issue by pritamghanghas** [NebraLtd/hm-diag#385]
Incorporate exponential error suppression as explained in this [comment](https://github.com/nebraltd/product-management/issues/67#issuecomment-1144655061)

**Acceptance Criteria**
- Exponential error suppression till a max of one error sent to sentry for every 128 errors

**Comment by marvinmarnold** [comment]
Let's keep this in QA until the Sentry allocation resets.

**Comment by marvinmarnold** [comment]
@pritamghanghas is the relevant code in production on helium-miner-software?

**Comment by KevinWassermann94** [comment]
@pritamghanghas ^

**Comment by pritamghanghas** [comment]
@KevinWassermann94  @marvinmarnold  Yes, it went with Network watchdog release. We will see how well it works out in supressing issue count and enable for other repos with a separate ticket.

**Comment by KevinWassermann94** [comment]
@pritamghanghas Watchdog hasn't been pushed to production yet
